### DISCUSSION TITLE
Thank You Gerganov and other contributors !!

### DISCUSSION THREAD

**Discussion by alimaslax** [ggml-org/whisper.cpp#656] [General]
I personally want to thank you guys for your work. It's people like you that make the world go round. The swiftUI demo is amazing, and fast. Whispers ability to pick up my voice in any tone and volume is aww inspiring.

You should write a book/summary of how you turned a pytorch program to pure C/C++. There are ALOT of AI models that need that same love. I.E CLIP. Python doesn't run well on all devices. And doesn't integrate well with other programming languages. But C does.

With the C API implementation I can add it to my Flutter Assistant App. I will share & tell my hackathon soon. Thank you, thank you, thank you.

**Comment by echogit** [comment]
Hi, did you managed to use whisper on android and ios? 
I found this flutter_whisper.cpp but it seems it is only for IOs, 
https://github.com/lyledean1/flutter_whisper.cpp
And also this article:
https://medium.com/bix-digital-engineers/integrating-whisper-model-into-ios-flutter-project-f2bcd881e7a0

But again only towards IOS
